### PR TITLE
Add SessionInvalidEvent support.

### PR DESCRIPTION
**Pull request by eyurko**

### CODE REVIEW

File: client/question/domain/ErrorTracebackObjectFactory.js [hunk 1]
```diff
@@ -94,6 +102,16 @@ tie.factory('ErrorTracebackObjectFactory', [
...
     };
 
+    /**
+     * Returns an ErrorTraceback object from a server error.
+     *
+     * @returns {ErrorTraceback}
+     */
+    ErrorTraceback.fromServerError = function() {
+      return ErrorTraceback.create(SERVER_ERROR_MESSAGE,
+        TracebackCoordinatesObjectFactory.create(0, 0));
```
[the remark this answers is not in the document]
Comment (eyurko) [reply client/question/domain/ErrorTracebackObjectFactory.js]: So I was wondering about that -- it doesn't super matter either way, but there's no handling for if the traceback coordinates are null, so I was trying to figure out if that was just not spec'ed or if it shouldn't happen.

Happy to add some light error checking if you'd prefer this to be null.

[the remark this answers is not in the document]
Comment (seanlip) [reply client/question/domain/ErrorTracebackObjectFactory.js]: Hm, thinking through what makes sense conceptually, I think null is better. So, if it doesn't mess other things up, I'd go with that if it works OK (and add explicit handling for the null case if necessary).

[the remark this answers is not in the document]
Comment (eyurko) [reply client/question/domain/ErrorTracebackObjectFactory.js]: That's fine -- I think I'll need to have a check for getFirstTracebackLine, but other than that, I should be fine. Do you mind unapproving this PR and I'll have you take another look once it's done (early tomorrow)?

[the remark this answers is not in the document]
Comment (seanlip) [reply client/question/domain/ErrorTracebackObjectFactory.js]: Sure! Thanks.

**Pull request merged**